dmoj/executors/mono_executor.py

```diff
@@ -32,7 +34,10 @@ class MonoExecutor(CompiledExecutor):
     # get flagged as MLE.
     data_grace = 65536
     cptbox_popen_class = MonoTracedPopen
-    fs = ['/etc/mono/']
+    fs = FilesystemPolicy([
+        RecursiveDir('/etc/mono/')
```

Comment:
Does having a trailing slash here work? `add_rule` doesn't seem to strip it, so the last part of the path will be `''`. There are also some other fs directory rules that don't have a trailing slash and we should probably make them consistent.

Reply:
Trailing slashes need to be removed everywhere they appear, yeah.

Reply:
We should assert that we have no empty components in the split path, to avoid reliance on human carefulness.